### ISSUE TITLE
Fails to install extensions on win7 32-bit

### ISSUE THREAD

**Issue by pronebird** [MarshallOfSound/electron-devtools-installer#72]
I run a windows 7 32-bit VM and I am not able to install the dev extensions, all it says it cannot fetch them, but I have internet for sure.

```
Failed to fetch extension, trying 4 more times
Failed to fetch extension, trying 3 more times
Failed to fetch extension, trying 2 more times
Failed to fetch extension, trying 1 more times
Failed to fetch extension, trying 0 more times
[2018-02-13 03:55:54.014][info] Error installing REACT_DEVELOPER_TOOLS extension
: spawn UNKNOWN
```

**Comment by vintproykt** [comment]
```
$ electron-forge start
√ Checking your system
√ Locating Application
√ Preparing native dependencies
√ Launching Application

Failed to fetch extension, trying 4 more times
Failed to fetch extension, trying 3 more times
Failed to fetch extension, trying 2 more times
Failed to fetch extension, trying 1 more times
Failed to fetch extension, trying 0 more times
(node:8200) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Err
or: spawn UNKNOWN
(node:8200) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the fu
ture, promise rejections that are not handled will terminate the Node.js process with a non-zero
 exit code.
```

We definitely need to find a workaround with @MarshallOfSound

**Comment by faizalMo** [comment]
We have come across same issue in Windows 7 32-bit. The extensions(react and redux) are not being installed. I think the issue is with unzipping the extension as I can see the extensions are downloaded into the extension folder, however installation fails with the following error.

`error: Error: spawn C:\Users\...\app-2.0.0\resources\app.asar\node_modules\7zip\7zip-lite\7z.exe ENOENT
      at _errnoException (util.js:1024:11)
      at Process.ChildProcess._handle.onexit (internal/child_process.js:190:19)
      at onErrorNT (internal/child_process.js:372:16)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)`

I have also noticed that if I manually unzip the .crx extension file and restarted the application, devtools are installed properly. @MarshallOfSound

**Comment by defusioner** [comment]
same problem here. Windows 10, 64-bit

**Comment by damienallen** [comment]
Spent the day migrating from electron-compile to webpack (and from electron-forge 5 to 6) and ran into this issue, also on Win10 64-bit. PR #92 works great for me.

**Comment by MarshallOfSound** [comment]
Fixed in #92
